test/youtex_test.exs

```diff
@@ -0,0 +1,89 @@
+defmodule YoutexTest do
+  use ExUnit.Case, async: true
+
+  use Youtex.Types
+  alias Youtex.Transcript
+  import Youtex
+
+  @en_only_video_id "guc16D_0Imk"  # ElixirConf 2020 - José Valim
+  @en_pt_video_id "lxYFOM3UJzo" # Elixir: The Documentary
```

Comment:
❤️

Reply:
Valim is a great inspiration for all the community :clap: